dspace-api/src/main/java/org/dspace/app/bulkedit/MetadataImport.java

```diff
@@ -652,14 +676,23 @@ private void handleRelationMetadata(Context c, Item item, String schema, String
 
     }
 
+    /**
+     * This method takes the item, element and values to determine what relationships should be built
+     * for these parameters and calls on the method to construct them
+     * @param c         The relevant DSpace context
+     * @param item      The item that the relationships will be made for
+     * @param element   The string determining which relationshiptype is to be used
+     * @param values    The value for the relationship
+     * @throws SQLException If something goes wrong
+     * @throws AuthorizeException   If something goes wrong
+     */
     private void handleRelationOtherMetadata(Context c, Item item, String element, List<String> values)
         throws SQLException, AuthorizeException {
         Entity entity = entityService.findByItemId(c, item.getID());
         boolean left = false;
         List<RelationshipType> acceptableRelationshipTypes = new LinkedList<>();
-        String[] components = values.get(0).split("-");
         String url = handleService.resolveToURL(c, values.get(0));
-        if (components.length != 5 && StringUtils.isNotBlank(url)) {
+        if (UUIDUtils.fromString(values.get(0)) == null && StringUtils.isNotBlank(url)) {
```

Comment:
Just a minor note: I would introduce a variable and use it here and 4 lines down to only convert the string once into a UUID.